ocs_ci/ocs/resources/cloud_manager.py

```diff
@@ -374,6 +390,7 @@ def internal_create_uls(self, name, region=None):
             self.client.create_bucket(name)
         else:
             self.client.create_bucket(name, location=region)
+        self.verify_uls_state(name, True)
```

Comment:
Please move the check to the `CloudClient` `create_uls` and `delete_uls` functions instead